Makefile.pre.in

```diff
@@ -267,6 +267,7 @@ STATIC_LIBPYTHON=	@STATIC_LIBPYTHON@
 
 LIBS=		@LIBS@
 LIBM=		@LIBM@
+LIBATOMIC=	@LIBATOMIC@
```

Comment:
I don't think this is needed; since we only need this for `_testcapi`, you'll be fine with `MODULE__TESTCAPI_LDFLAGS` (it is set automatically via `PY_STDLIB_MOD` in `configure.ac`.

Reply:
Shortly, LIBATOMIC will be needed by more code. I'm pretty sure that @colesbury will use <cpython/pyatomic.h> in a few more C files :-D

Moreover, I'm trying to generate ``MODULE__TESTCAPI_LDFLAGS=$(LIBATOMIC)`` in the Makefile. So after configure, it would remind possible to "fix" LIBATOMIC once for all object files which use the variable.

But I failed to fail a syntax to generate this line, configure strongly wants to replace LIBATOMIC variable with its value :-)

Reply:
I think this should be done in `configure.ac`. If it is needed by other extension modules, we add it to their `PY_STDLIB_MOD`. If it is needed by everything (a global linker option), we can use one of the `LDFLAGS` variables that are already in place :)

Reply:
I disagree, see my comments on the PR below.